src/test/kotlin/org/jetbrains/research/ml/ast/transformations/util/TransformationsTestHelper.kt

```diff
@@ -50,9 +46,9 @@ object TransformationsTestHelper {
     fun assertCodeTransformation(
         inFile: File,
         outFile: File,
-        transformation: (PsiElement, Boolean) -> Unit,
+        transformation: TransformationDelayed,
```

Comment:
Do we really need to have CommandStorage as an argument in TransformationDelayed? Because we always call this function (assertCodeTransformation) with cs = null, since we deal with the correct cs value in wrappers. So maybe we can change the TransformationDelayed signature? And also remove commandStorage argument from getForwardTransformationWrapper since it's always null